generator/lib/src/entity_resolver.dart

```diff
@@ -153,7 +153,9 @@ class EntityResolver extends Builder {
               "  invalid relation property '${f.name}' in entity '${element.name}' - must use ToOne/ToMany<TargetEntity>");
           continue;
         }
-        relTargetName = (f.type as ParameterizedType).typeArguments[0].name;
+        relTargetName = (f.type as ParameterizedType)
+            .typeArguments[0]
+            .getDisplayString(withNullability: false);
```

Comment:
The docs say "Clients should not depend on the content of the returned value as it will be changed if doing so would improve the UX.". But this string is used in e.g. the code generator, no? Is this a good idea then?

Reply:
`.name` got deprecated... I misread the suggestion, I was supposed to use *either* `element` or `getDisplayString()`... `element` is the way to go so changed it.